server/services/applicationService.ts

```diff
@@ -56,16 +53,6 @@ export default class ApplicationService {
     return page
   }
 
-  getResponses(application: Application): ApplicationResponse {
```

Comment:
We will need this going forward, it's just not used anywhere _yet_

Reply:
Ha, ok!